tssc/step_implementers/uat/maven.py

```diff
@@ -150,21 +156,32 @@ def _get_target_base_url(self):
 
         return target_base_url
 
+    def _settings_file(self):
```

Comment:
@ppremru @akim309 suggest calling this `__generate_maven_settings` for consistancy.

Reply:
updated